daphne_worker/src/lib.rs

```diff
@@ -352,17 +352,42 @@ impl DaphneWorkerRouter {
                     |mut req, ctx| async move {
                         let config = DaphneWorkerConfig::from_worker_context(ctx)?;
                         let cmd: InternalTestEndpointForTask = req.json().await?;
-                        config.internal_endpoint_for_task(cmd).await
+                        config
+                            .internal_endpoint_for_task(config.default_version, cmd)
+                            .await
+                    },
+                )
+                .post_async(
+                    "/:version/internal/test/endpoint_for_task",
```

Comment:
Why not replace the old endpoint with the new one? The advantage of having one is less code. (Similarly below.)

Reply:
I was trying to not break any code relaying on the old endpoint, just in case